.drone.yml

```diff
@@ -99,6 +76,25 @@ steps:
     - success
     - failure
 
+---
+
```

Comment:
I don't think we need this? Although a `kube-hocs-serenity-automation` repo exists, we don't seem to use it? We don't need to host the app in our environments as a pod. If we don't need it, remove all of this green chunk.

Reply:
Removed